workspaces/ui-v2/src/pages/docs/DocumentationRootPage/EndpointRow.tsx

```diff
@@ -94,7 +108,10 @@ export const EndpointRow: FC<EndpointRowProps> = ({ endpoint }) => {
         >
           <EndpointNameMiniContribution
             id={endpointId}
-            endpointId={endpointId}
+            endpoint={{
```

Comment:
I had to change this interface since we want to specifically split out method and pathId when storing these in contributions and in the deleted endpoint state.